daml-assistant/daml-helper/BUILD.bazel

```diff
@@ -44,10 +44,13 @@ da_haskell_library(
     visibility = ["//visibility:public"],
     deps = [
         "//compiler/daml-lf-ast",
+        "//compiler/daml-lf-proto",
         "//compiler/damlc/daml-compiler",
```

Comment:
I'll check what fails when I remove it. But the `DA.Daml.Compiler.Dar` module is definitely a problem. Agreed that longterm we should aim at removing it from the compiler library.

Reply:
One of the reason why we depend on the compiler is `fetch-dar`. We had a plan to allow fetching DARs automatically if you specify a package id in your daml.yaml and for that it would be convenient if the code was in the compiler and you didn’t have to shell out to `daml-helper`. But we never implemented this so it didn’t go anywhere.

Reply:
Yes, `daml-helper` depends on it because of `fetch-dar`.